pkg/apis/org/v1/che_types.go

```diff
@@ -225,67 +203,53 @@ type CheClusterSpecServer struct {
+	// and no additional configuration is required, but defining `proxyUrl` in a custom resource leads to overrides the cluster proxy configuration
+	// with fields `proxyUrl`, `proxyPort`, `proxyUser` and `proxyPassword` from the custom resource.
+	// (see the doc https://docs.openshift.com/container-platform/4.4/networking/enable-cluster-wide-proxy.html) (see also the `proxyPort` and `nonProxyHosts` fields).
 	// +optional
 	ProxyURL string `json:"proxyURL,omitempty"`
-	// Port of the proxy server. Only use when configuring a proxy is required.
-	// (see also the `proxyURL` and `nonProxyHosts` fields).
+	// Port of the proxy server. Only use when configuring a proxy is required. (see also the `proxyURL` and `nonProxyHosts` fields).
 	// +optional
 	ProxyPort string `json:"proxyPort,omitempty"`
-	// List of hosts that should not use the configured proxy.
-	// So specify wild card domain use the following form `.<DOMAIN>` and `|` as delimiter, eg: `localhost|.my.host.com|123.42.12.32`
-	// Only use when configuring a proxy is required.
-	// Operator respects OpenShift cluster wide proxy configuration and no additional configuration is required,
-	// but defining `nonProxyHosts` in a custom resource leads to merging non proxy hosts lists from the
-	// cluster proxy configuration and ones defined in the custom resources.
-	// (see the doc https://docs.openshift.com/container-platform/4.4/networking/enable-cluster-wide-proxy.html)
-	// (see also the `proxyURL` fields).
+	// List of hosts that should be reached directly, bypassing the proxy.
+	// Specify wild card domain use the following form `.<DOMAIN>` and `|` as delimiter, for example: `localhost|.my.host.com|123.42.12.32`
+	// Only use when configuring a proxy is required. Operator respects OpenShift cluster wide proxy configuration and no additional configuration is required,
+	// but defining `nonProxyHosts` in a custom resource leads to merging non proxy hosts lists from the cluster proxy configuration and ones defined in the custom resources.
+	// (see the doc https://docs.openshift.com/container-platform/4.4/networking/enable-cluster-wide-proxy.html) (see also the `proxyURL` fields).
 	NonProxyHosts string `json:"nonProxyHosts,omitempty"`
-	// User name of the proxy server.
-	// Only use when configuring a proxy is required
-	// (see also the `proxyURL`, `proxyPassword` and `proxySecret` fields).
+	// User name of the proxy server. Only use when configuring a proxy is required (see also the `proxyURL`, `proxyPassword` and `proxySecret` fields).
 	// +optional
 	ProxyUser string `json:"proxyUser,omitempty"`
-	// Password of the proxy server
-	// Only use when proxy configuration is required
-	// (see also the `proxyURL`, `proxyUser` and `proxySecret` fields).
+	// Password of the proxy server Only use when proxy configuration is required (see also the `proxyURL`, `proxyUser` and `proxySecret` fields).
```

Comment:
> Password of the proxy server

Should be '.' in the end of the sentences?

Reply:
right.